squape/video.py

```diff
@@ -63,7 +63,7 @@ def _replace_videos(videos: set) -> None:
 
     Args:
         videos (set): A set of video names to replace.
-        Does nothing when the set is empty.
+            Does nothing when the set is empty.
```

Comment:
Will those changes be present in the sources too?

Reply:
I don't understand the question.